src/actions/app.js

```diff
@@ -161,6 +171,37 @@ export const saveExisting = () => (dispatch, getState) => {
       });
 };
 
+export const deleteSnippet = snippetKey => (dispatch, getState) => {
+  // Get items out of app state
+  const {
+    user: {
+      token,
+      selectedOrg,
+    },
+  } = getState();
+
+  // Construct request objects
+  const reqHeaders = {
+    headers: {
+      Authorization: token,
+    },
+  };
+
+  // Delete the snippet
+  dispatch(addNotification('Deleting...'));
+  return axios.delete(makeSaveEndpointUrl(selectedOrg, snippetKey), reqHeaders)
+    .then(() => {
+      // Remove the 'deleting...' notifications
+      dispatch(closeNotification());
+      // Give user feedback that snippet deleted
+      dispatch(addNotification('Snippet Deleted!'));
+    })
+    .catch((err) => {
+      dispatch(addNotification('Failed to delete snippet; please try again'));
```

Comment:
You would probably want to dispatch a `closeNotification()` call here, so that this notification is presented faster.